### PR TITLE
[HSEARCH] 7.0.0.CR1

### PR DESCRIPTION
**Pull request by yrodiere**
To be merged after review, ideally this Monday

### CODE REVIEW

**Comment by yrodiere** [comment]
Staging will be available shortly at https://staging.in.relation.to/2023/10/23/hibernate-search-7-0-0-CR1/

File: posts/Yoann/2023-10-23-hibernate-search-7-0-0-CR1.adoc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-version-family: 7.0
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id: 32189
+---
+
+We just published a new candidate release for Hibernate Search: 7.0.0.CR1.
+
+This release brings in particular compatibility with Hibernate ORM's discriminator-based multi-tenancy,
+compatibility with Elasticsearch 8.10 and OpenSearch 2.9/2.10,
```
Comment (marko-bekhta) [inline posts/Yoann/2023-10-23-hibernate-search-7-0-0-CR1.adoc]:
```suggestion
compatibility with Elasticsearch 8.10 and OpenSearch 2.9/2.10/2.11,
```

**Review by marko-bekhta — APPROVED** [review]
All good 😃 only that OpenSearch version... maybe it was supposed to be `2.10/2.11` ?

**Comment by yrodiere** [comment]
Good catch, thanks, I applied your suggestion. Let's merge this on Monday :)

**Pull request merged**